server/routes/creatingLicences/types/additionalConditionInputs/outOfBoundsRegion.ts

```diff
@@ -3,8 +3,12 @@ import { IsNotEmpty } from 'class-validator'
 
 class OutOfBoundsRegion {
   @Expose()
-  @IsNotEmpty({ message: 'Enter the name of the area on the map' })
+  @IsNotEmpty({ message: 'Enter the name of the area shown in the attached map' })
   outOfBoundArea: string
+
+  @Expose()
+  @IsNotEmpty({ message: 'You must attach an exclusion zone MapMaker PDF file' })
+  outOfBoundFilename: string
```

Comment:
Could annotate this with a custom validator which checks that `req.file` contains the correct things which match this field

Reply:
Good suggestion - looking now.

Reply:
Custom validator is now used.